### PR TITLE
Final tweaks for upcoming events

### PR DESCRIPTION
**Pull request by toddr**

### CODE REVIEW

File: docs/www/events.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [%-
   page.import({
-      title    => 'Perl events',
+      title    => 'Perl and Raku events',
```
Comment (rspier) [inline docs/www/events.html]: Please drop 'and Raku' from the title and description.

Comment (toddr) [reply docs/www/events.html]: That was the agreed-upon language with the TPF president

Comment (rspier) [reply docs/www/events.html]: TPF doesn't run this site. I'm just asking for the page level content, not the event names.

Comment (toddr) [reply docs/www/events.html]: The events covered relate to both Perl and Raku events. Why would that be inappropriate?

Comment (toddr) [reply docs/www/events.html]: I'm going to remove 479020772a522f7260b204eb98ade471b23c0432 from this PR so we can get the critical stuff in. I'll open a separate PR for Raku issues.

**Comment by rspier** [comment]
Merged, will go live soon.

**Pull request merged**